### ISSUE TITLE
[Fix] PCRData.positive_rate() fails line plot when color is not specified

### ISSUE THREAD

**Issue by lisphilar** [lisphilar/covid19-sir#521]
## Summary
Line plotting fails when color is not specified. E.g. `pcr_data.positive_rate()`
This is related to #481 and this occurs with some versions of pandas/matplotlib.

## Codes
```Python
import covsirphy as cs
# Dataset preparation
data_loader = cs.DataLoader("input")
pcr_data.positive_rate(country)
```

## Outputs
Raises `ValueError: Invalid color None` with covsirphy/util/plotting.py.

## Environment
- CovsirPhy version: 2.14.0-delta with pull request #520 
- Python version: 3.8.5
- Installation: pip
- System: Linux

**Comment by lisphilar** [comment]
#522 was merged to version 2.14.0-epsilon.

**Comment by Inglezos** [comment]
I try to run snl.records(variables=["Confirmed", "Infected", "Fatal", "Recovered"], color_dict={"Confirmed": "blue", "Infected": "orange", "Fatal": "red", "Recovered": "green"}) but now it fails. Have you changed something in the way we called that?

**Comment by lisphilar** [comment]
Oh sorry, I fixed the error with #524.
